{"name": "Login with correct email and password", "status": "broken", "statusDetails": {"message": "AttributeError: module 'tools.logger' has no attribute 'info'", "trace": "public_users_client = <clients.users.public_users_client.PublicUsersClient object at 0x119ef2b40>\n\n    @pytest.fixture\n    def function_user(public_users_client: PublicUsersClient) -> UserFixture:\n        request = CreateUserRequestSchema()\n>       response = public_users_client.create_user(request)\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nfixtures/users.py:40: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nclients/users/public_users_client.py:24: in create_user\n    response = self.create_user_api(request)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nclients/users/public_users_client.py:21: in create_user_api\n    return self.post(\"/api/v1/users\", json=request.model_dump(by_alias=True))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\nclients/api_client.py:41: in post\n    return self.client.post(url, json=json, data=data, files=files)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n../../projects/bot-python/.venv/lib/python3.12/site-packages/httpx/_client.py:1144: in post\n    return self.request(\n../../projects/bot-python/.venv/lib/python3.12/site-packages/httpx/_client.py:825: in request\n    return self.send(request, auth=auth, follow_redirects=follow_redirects)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n../../projects/bot-python/.venv/lib/python3.12/site-packages/httpx/_client.py:914: in send\n    response = self._send_handling_auth(\n../../projects/bot-python/.venv/lib/python3.12/site-packages/httpx/_client.py:942: in _send_handling_auth\n    response = self._send_handling_redirects(\n../../projects/bot-python/.venv/lib/python3.12/site-packages/httpx/_client.py:977: in _send_handling_redirects\n    hook(request)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nrequest = <Request('POST', 'http://localhost:8000/api/v1/users')>\n\n    def log_request_event_hook(request: Request):  # Создаем event hook для логирования запроса\n        \"\"\"\n        Логирует информацию об отправленном HTTP-запросе.\n    \n        :param request: Объект запроса HTTPX.\n        \"\"\"\n        # Пишем в лог информационное сообщение о запроса\n>       logger.info(f'Make {request.method} request to {request.url}')\n        ^^^^^^^^^^^\nE       AttributeError: module 'tools.logger' has no attribute 'info'\n\nclients/event_hooks.py:27: AttributeError"}, "start": 1752323559055, "stop": 1752323559055, "uuid": "5eb612fc-7311-4f0c-8b56-84cc4f998e33", "historyId": "b6b2aaa730ad45068d1fca8abf4a80e6", "testCaseId": "b6b2aaa730ad45068d1fca8abf4a80e6", "fullName": "tests.authentication.test_authentication.TestAuthentication#test_login", "labels": [{"name": "feature", "value": "Authentication"}, {"name": "severity", "value": "blocker"}, {"name": "epic", "value": "LMS service"}, {"name": "tag", "value": "REGRESSION"}, {"name": "story", "value": "Login"}, {"name": "tag", "value": "AUTHENTICATION"}, {"name": "tag", "value": "authentication"}, {"name": "tag", "value": "regression"}, {"name": "parentSuite", "value": "tests.authentication"}, {"name": "suite", "value": "test_authentication"}, {"name": "subSuite", "value": "TestAuthentication"}, {"name": "host", "value": "VIS-SSQ05N"}, {"name": "thread", "value": "77586-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.authentication.test_authentication"}]}